Atomic/Export.py

```diff
@@ -0,0 +1,107 @@
+"""
+export docker images, containers and volumes into a filesystem directory.
+"""
+import sys
+import os
+import subprocess
+import Atomic.util as util
+
+def export_docker(graph, export_location):
+    """
+    This is a wrapper function for exporting docker images, containers
+    and volumes.
+    """
+
+    if os.geteuid() != 0:
+        exit("You need to have root privileges to run atomic export."
+             "\nPlease try again, this time using 'sudo'. Exiting.")
+
+    if not os.path.isdir(export_location):
+        os.mkdir(export_location)
+    try:
+	#Save the docker storage driver
+	storage_driver = subprocess.check_output("docker info|grep \"Storage Driver\"|cut -d\" \" -f 3", shell=True)
+	file = open(export_location+"/dockerInfo.txt", "w")
```

Comment:
d = docker.Client()
Use self.d defined in Atomic to use docker.py
d.info()["Driver"]
u'devicemapper'